config.md

```diff
@@ -249,7 +249,7 @@
 
 |Key|Description|Type|Default Value|
 |---|-----------|----|-------------|
-|factor|The retry backoff factor|`boolean`|`2`
+|factor|The retry backoff factor|`int`|`2`
```

Comment:
This will be fixed by the same `firefly-common` fix, once worked out.

Reply:
https://github.com/hyperledger/firefly-common/pull/36